ui/store/action-queue/index.js

```diff
@@ -0,0 +1,135 @@
+import pify from 'pify';
```

Comment:
It doesn't look like we need the Proxy based heavy lifting this pkg is doing.

Reply:
I agree

Reply:
I agree with you point here, but I would prefer to create a separate task for removing `pify`.